### DISCUSSION TITLE
Iris like a mod

### DISCUSSION THREAD

**Discussion by PandaWolf39** [IrisShaders/Iris#1404] [Q&A] [answered]
Hello,
(sorry if my english is bad, i speak with google translate ._.)
I would like to know if Iris can or will be used as a mod with Forge/Fabric, as Optifine can because it will allow me to use some mods as I am used to.
Thanks in advance,
PandaWolf
<img width="393" alt="Capture d’écran 2022-05-01 à 18 14 33" src="https://user-images.githubusercontent.com/104728631/166154847-8d115d69-af81-46fc-912b-50c887fe9530.png">

**Comment by altrisi** [comment, marked as answer]
Yes, Iris is a Fabric mod. You can either use the installer to install it as a mod (by running it and clicking "install as a fabric mod") or you can download it as a mod jar on modrinth, curseforge or github releases.

**Comment by PandaWolf39** [comment]
thank you very much
